### ISSUE TITLE
Fails to import resource

### ISSUE THREAD

**Issue by jkdey** [terraform-providers/terraform-provider-launchdarkly#19]
I get the following error when trying to import resources in 1.1.0 and 1.0.1:

```
terraform import launchdarkly_feature_flag.video-cache search/video-cache
Acquiring state lock. This may take a few moments...
launchdarkly_feature_flag.video-cache: Importing from ID "search/video-cache"...

Error: nil entry in ImportState results. This is always a bug with
the resource that is being imported. Please report this as
a bug to Terraform.
```

**Comment by ldhenry** [comment]
Hey @jdkey,

Thanks for bringing this to my attention. After some investigation, I uncovered a bug in the import validation for this resource.

I was able to reproduce your error when I tried to run a `terraform import` with an invalid flag ID (`project_key/flag_key`). However, there is still a chance that there could be something else going on.

To help me narrow this down, please double check that `search` and `video-cache` are indeed the correct project and flag keys (not names) and report back here.

Thanks,
Henry

**Comment by ldhenry** [comment]
Hi @jkdey,

I just released v1.2.0 of the provider which has a bug fix for the import validation. Let me know if this works for you and feel free to re-open this issue if you are still experiencing trouble.

Thanks,
Henry

**Comment by jkdey** [comment]
This updated provider fixed the issue and appropriately represented a clearer error pointing at an incorrectly specified key. Thanks!
